plugins/woocommerce-admin/client/customer-effort-score-tracks/data/actions.js

```diff
@@ -49,6 +49,41 @@ export function addCesSurvey(
 	};
 }
 
+/**
+ * Add show CES modal.
+ *
+ * @param {string} action              action name for the survey
+ * @param {string} label               label for the snackback
+ * @param {string} onsubmitLabel       label for the snackback onsubmit
+ * @param {Object} props               object for optional props
+ * @param {Object} onSubmitNoticeProps object for on submit notice props.
+ */
+export function showCesModal(
```

Comment:
Since this function is accepting a number of arguments, some of which could be undefined, perhaps we should have this accept an object with optional properties instead?

```js
export function showCesModal( {
	action,
	label,
	onsubmitLabel = undefined,
	props = {},
	onSubmitNoticeProps = {}
} ) {
	return {
```